truffle-config.js

```diff
@@ -13,24 +6,15 @@ module.exports = {
       port: 8545,
       network_id: '*', // Match any network id
       gasPrice: 10000000000,
-      gas: 7000000
```

Comment:
how come youve taken the gas down to 4800000?

Reply:
oh you answered this in your description

Reply:
4,800,000 was an older default gas settings with a vanilla setup of ganache, however I can increase it to 6700000 which looks like the new default

Reply:
Yeah lets increase to 6.7m please 👍